cli/uninstall/uninstall.go

```diff
@@ -187,6 +187,11 @@ func GetList(cliOpts *config.CliOpts, program string) []string {
 	return list
 }
 
+// checkHashFormat checks hash format.
+func checkHashFormat(hash string) (bool, error) {
```

Comment:
We have the same function in the `install` package. Why don't move it to common package? For example move to `git` package in `util` directory and name it `isValidCommitHash`. And what about length? Don't we need to check it? Suppose, 'aaa' is not a valid hash.

Reply:
fixed

Reply:
I can't add length check in this function, because in install need "at least 7 digits" and in uninstall "only 7 digits"